### ISSUE TITLE
typings or tsconfig problem?

### ISSUE THREAD

**Issue by cbritton** [alexjoverm/Vue-Typescript-Starter#8]
Getting this list of issues when starting,  I'm no expert on typescript so I have no idea what to make of this.

`$ node build/dev-server.js
> Starting dev server...
ts-loader: Using typescript@2.3.2 and F:\Chris\workspace\vue-test\Vue-Typescript-Starter\tsconfig.json

 ERROR  Failed to compile with 7 errors                                                                                                                                               11:26:54 AM

 error  in F:\Chris\workspace\vue-test\Vue-Typescript-Starter\node_modules\vue-router\types\router.d.ts
(4,35): error TS2304: Cannot find name 'Vue'.
 error  in F:\Chris\workspace\vue-test\Vue-Typescript-Starter\node_modules\vue-router\types\router.d.ts
(13,43): error TS2304: Cannot find name 'Vue'.
 error  in F:\Chris\workspace\vue-test\Vue-Typescript-Starter\node_modules\vue-router\types\router.d.ts
(19,8): error TS2304: Cannot find name 'Vue'.
 error  in F:\Chris\workspace\vue-test\Vue-Typescript-Starter\node_modules\vue-router\types\router.d.ts
(93,25): error TS2304: Cannot find name 'Vue'.
 error  in F:\Chris\workspace\vue-test\Vue-Typescript-Starter\node_modules\vue-router\types\vue.d.ts
(17,13): error TS2428: All declarations of 'ComponentOptions' must have identical type parameters.
 error  in F:\Chris\workspace\vue-test\Vue-Typescript-Starter\node_modules\vue-router\types\vue.d.ts
(17,40): error TS2304: Cannot find name 'Vue'.
 error  in F:\Chris\workspace\vue-test\Vue-Typescript-Starter\node_modules\vue\types\options.d.ts
(54,18): error TS2428: All declarations of 'ComponentOptions' must have identical type parameters.
> Listening at http://localhost:8080`

**Comment by cbritton** [comment]
I managed to get this working.  The problem seems to be with the newer version of Vue (2.5.2).  Because package.json uses the caret(^) on vue and vue-router, npm/yarn will get the latest. To not use the latest vue, remove the ^ in the package.json file for vue, vue-router and vue-template-compiler, like so:

`    "vue": "2.2.6",
    "vue-router": "2.3.1"
`
...
`
    "vue-template-compiler": "2.2.6",
`
I removed node_modules for good measure, then npm/yarn install.  No errors on starting. I also tried this with typescript 2.5.3 and it works.  Whatever is going on with Vue 2.5.3 with those errors is beyond me.
